src/components/opinion/opinionCutout.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import { css, SerializedStyles } from '@emotion/core';
+import { Contributor, isSingleContributor } from 'capi';
+import { transformUrl } from 'asset';
+
+const imageWidth = 68;
+
+const OpinionCutoutStyles = css`
+    position: relative;
+`;
+
+const ImageStyles = css`
+    position: absolute;
+    height: 160px;
+    right: 0;
+    top: -54px;
+`;
+
+interface OpinionCutoutProps {
+    contributors: Contributor[];
+    imageSalt: string;
+    className: SerializedStyles;
+}
+
+const OpinionCutout = ({
+    contributors,
+    imageSalt,
+    className
+}: OpinionCutoutProps): JSX.Element | null => {
+    const [contributor] = contributors;
+
+    if (isSingleContributor(contributors) && contributor.bylineLargeImageUrl) {
+        const imgSrc = transformUrl(imageSalt, contributor.bylineLargeImageUrl, imageWidth*3);
+        return (
+            <div css={[className, OpinionCutoutStyles]}>
+                <img css={ImageStyles} src={imgSrc} alt={contributor.webTitle}/>
+            </div>
+        );
```

Comment:
Is there any way we can use some of the functions in `image` or `asset` to make this a `srcset`?

Reply:
hmm I don't think we get the Asset data type. Also I don't think the image changes size on different screen widths.

Reply:
This is the author avatar? Yeah I guess that would stay the same "css size", but I was thinking more about [resolution switching](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images#Resolution_switching_Same_size_different_resolutions).